### ISSUE TITLE
[Anathema] Green wands have too much +dmg/healing

### ISSUE THREAD

**Issue by Lorilay2** [elysium-project/server#1077]
After 1.9, there was a nerf to green +dmg/healing stat allocations. Patch notes:
> Incorrect amounts of abilities such as Attack Power, Increased Fire Damage, or Increased Healing Damage were being applied to all random item property world drops. Two handed weapons such as Staves had much lower amounts of these abilities, while ranged and one handed weapons had too much. All further drops of these items will have the correct amounts, but previously dropped items will not be affected by this change.

The current item allocations on Anathema use the pre-nerfed versions (+30ish healing, +20ish damage). They should be updated to the nerfed values (+11 damage, +20 healing).

**More Evidence:**
[August 2005](https://web.archive.org/web/20050829052852/http://wow.allakhazam.com:80/db/item.html?witem=15281): No +11 damage (pre-nerf)
[February 2006](https://web.archive.org/web/20060216103712/http://wow.allakhazam.com:80/db/item.html?witem=15281): +11 damage exists (previously dropped items still in game, though)

**Comment by Spaztick** [comment]
Itemization is currently 1.9, talents are 1.12.

**Comment by taurenrogue** [comment]
> All further drops of these items will have the correct amounts, but previously dropped items will not be affected by this change.
the existing items should **not** be nerfed but new ones should not be dropping anymore with high rolls

**Comment by Lorilay2** [comment]
Yes, and yesterday a +33 healing wand dropped, hence this bug report.
